demo/body3d_two_stage_video_demo.py

```diff
@@ -0,0 +1,357 @@
+import copy
+import os
+import os.path as osp
+from argparse import ArgumentParser
+
+import cv2
+import mmcv
+import numpy as np
+
+from mmpose.apis import (get_track_id, inference_pose_lifter_model,
+                         inference_top_down_pose_model, vis_3d_pose_result)
+from mmpose.apis.inference import init_pose_model
+
+try:
+    from mmdet.apis import inference_detector, init_detector
+
+    has_mmdet = True
+except (ImportError, ModuleNotFoundError):
+    has_mmdet = False
+
+
+def process_mmdet_results(mmdet_results, cat_id=1):
+    """Process mmdet results, and return a list of bboxes.
+
+    Args:
+        mmdet_results (list|tuple): mmdet results.
+        cat_id (int): category id (default: 1 for human)
+
+    Returns:
+        person_results (list): a list of detected bounding boxes
+    """
+    if isinstance(mmdet_results, tuple):
+        det_results = mmdet_results[0]
+    else:
+        det_results = mmdet_results
+
+    bboxes = det_results[cat_id - 1]
+
+    person_results = []
+    for bbox in bboxes:
+        person = {}
+        person['bbox'] = bbox
+        person_results.append(person)
+
+    return person_results
+
+
+def dataset_transform(keypoints, pose_det_dataset, pose_lift_dataset):
```

Comment:
Can we use a more clear function name like "convert_keypoint_format" or "convert_keypoint_definition"?